model/src/main/java/oracle/kubernetes/weblogic/domain/model/DomainSpec.java

```diff
@@ -99,6 +99,21 @@
   @Description("If true (the default), the server .out file will be included in the pod's stdout.")
   private Boolean includeServerOutInPodLog;
 
+  /**
+   * Whether to fail to WebLogic server boot if any situational configuration error occurs during
+   * server startup. Default is true.
+   *
+   * @since 2.3.0
+   */
+  @Description(
+      "If true (the default), on WebLogic server that supports this feature, the WebLogic server "
+          + " boot would fail if any errors occur when applying situational configuration "
+          + " during server startup."
+          + " If false, WebLogic server would start if there are errors in the"
+          + " situational configuration files, but some configuration overrides may"
+          + " be skipped.")
+  private Boolean failBootOnSituationalError;
+
```

Comment:
'any errors' is promising too much - the setting doesn't capture all errors.   There are multiple errors that aren't captured -- you mispell an attribute name (IIRC), or reference an attribute that isn't there, or use 'add' when 'replace' was needed, etc, etc.    

How about:
 "If true (the default), on a WebLogic server that supports this feature, the WebLogic server "
	          + " boot will fail if it detects errors when applying situational configuration "
	          + " during server startup, as may happen if incorrectly configuration overrides templates"
                  + " are specified (see configOverrides attributes)."
	          + " If false, WebLogic server will still start even if there are errors detected in the"
	          + " situational configuration files but some or all configuration overrides may"
	          + " be skipped.")

Reply:
Similarly, the phrasing 'any situational configuration files are invalid' and similar phrasing that's used in other places within the change also implies to me an overly strong guarantee that WL will detect all situational configuration errors.  Not sure how to word it though.   Maybe something along the lines of 'common situational configuration file problems' 'most situational configuration file problems', etc...

Reply:
How about using something along the line of the actual WebLogic messages that were logged while loading sit config files:
BEA-141322 - The situational config file has an invalid format
BEA-141329 - Error loading the situational config files
So something along the line of "..the WebLogic server boot will fail if it encounters an error while loading the situational configuration files, or if a situational configuration file has an invalid format"
I don't know if that is better, but at least what we document here would be consistent with the error messages customers will find in the WebLogic log.

Reply:
Sure.  Anything that implies 'detects most errors' as opposed to 'all errors' or 'any errors'.

Reply:
This is outdated since the Domain spec attribute has been removed. I have applied the suggestion to the changes in configoverrides doc.